src/data/real/cau_seq_completion.lean

```diff
@@ -119,11 +115,22 @@ congr_arg mk (const_sub _ _)
 
 end
 
+section
+parameters {α : Type*} [linear_ordered_field α]
```

Comment:
I never use `parameters` so I don't understand it.

Aside from the weirdness that `parameters` can only be used inside a section, I don't know how it differs from `variables`.

What happens if you try to use `variables` here? What is the reason for using `parameters` in this file?

Reply:
This whole file uses `parameters`, so I figured I would just emulate it. I suspect the local notation makes it necessary.